### PR TITLE
e2e: fix deletion of persisten volumes

### PR DESCRIPTION
**Pull request by katexochen**
Signed-off-by: Paul Meyer <49727155+katexochen@users.noreply.github.com>

### Proposed change(s)

- Deletion of persistent volumes failed in case where there wasn't a kubeconfig, and termination of Constellation didn't happen. Fixed by continuing on error and checking input. See https://github.com/edgelesssys/constellation/actions/runs/3413360699/jobs/5679984053
- Delete PV deletion in k-bench payload, so it's not done twice.

### CODE REVIEW

**Comment by katexochen** [comment]
Test run k-bench: https://github.com/edgelesssys/constellation/actions/runs/3419141267/jobs/5692284746

**Review by chmey — APPROVED** [review]
Your code LGTM.

I the deletion of namespaces that contain PVCs

image

**Comment by katexochen** [comment]
@chmey do you have a test run for your last commit?

**Comment by chmey** [comment]
> @chmey do you have a test run for your last commit?

Just started:


- nop: https://github.com/edgelesssys/constellation/actions/runs/3422057978
- k-bench: https://github.com/edgelesssys/constellation/actions/runs/3422060326

**Comment by chmey** [comment]
Moved the debug `exit` step to simulate a case that would require clean up (pod with bound pvc).

That run is here: https://github.com/edgelesssys/constellation/actions/runs/3423648611

Should fail in K-Bench but then be cleaned up and terminate correctly.

File: .github/actions/k-bench/action.yml [hunk 1]
```diff
@@ -52,6 +52,7 @@ runs:
         kubectl create namespace kbench-pod-namespace --dry-run=client -o yaml |  kubectl apply -f -
         kubectl apply -f ../.github/actions/k-bench/encrypted_storage.yml
         ./run.sh -r "kbench-constellation-${{ inputs.cloudProvider }}" -t "dp_fio" -o "./out/"
+        exit 1
```
Comment (chmey) [inline .github/actions/k-bench/action.yml:55]: remove debug instruction before merge

**Review by chmey — APPROVED** [review]

**Pull request merged**